app/services/mhv_accounts_service.rb

```diff
@@ -94,13 +94,13 @@ def handle_upgrade!
   end
 
   def address_params
-    if user.va_profile&.address.present?
+    if user.address.present?
       {
-        address1: user.va_profile.address.street,
-        city: user.va_profile.address.city,
-        state: user.va_profile.address.state,
-        zip: user.va_profile.address.postal_code,
-        country: user.va_profile.address.country
+        address1: user.address.street,
```

Comment:
So annoying that this one address hash has `address1`, could have had the same address hash the same everywhere if it weren't for that, haha